components/dashtable_v2.js

```diff
@@ -70,8 +70,8 @@ export default function GlobalDashboardTable(props) {
         <a
           // TODO:
           href={
-            `/dashboard/v2/details/clh3ppzvg0001br794kgbw39a/` +
-            `${studentName}`
+            `/dashboard/v2/details/clh3ppzvg0001br794kgbw39a/ + ${userID}/ +
```

Comment:
My apologies for the confusion. We actually don't want to use the userID. Instead, we want to use the classroom ID for the url. We have updated the [description of the issue](https://github.com/freeCodeCamp/classroom/issues/320). Could you please take a look?

Also, the "clh3ppzvg0001br794kgbw39a" should be removed. Please take a look at the issue for a concrete example.